bndtools.m2e/bnd.bnd

```diff
@@ -36,10 +37,11 @@ Bundle-SymbolicName: bndtools.m2e; singleton:=true
 Bundle-ActivationPolicy: lazy
 
 Import-Package: \
- org.apache.maven.*;bundle-symbolic-name="${@bundlesymbolicname}";bundle-version="${range;[==,+);${@bundleversion}}";version=!,\
- org.codehaus.plexus.*;bundle-symbolic-name="${@bundlesymbolicname}";bundle-version="${range;[==,+);${@bundleversion}}";version=!,\
- ${eclipse.importpackage},\
- *
+	org.apache.maven.*;bundle-symbolic-name="${@bundlesymbolicname}";bundle-version="${range;[==,+);${@bundleversion}}";version=!,\
+	org.codehaus.plexus.*;bundle-symbolic-name="${@bundlesymbolicname}";bundle-version="${range;[==,+);${@bundleversion}}";version=!,\
+	${eclipse.importpackage},\
+	bndtools.launch,\
```

Comment:
Why isn't `bndtools.launch` found by the `*` entry? It seems wrong to hard code this. Is this because it is only referenced in the plugin.xml? If so, then a comment is needed here so someone tidying up later does not delete the line assuming it is covered by `*`. Perhaps a better idea is to put some reference (along with a comment why) to bndtools.launch in an m2e class so that Bnd will import bndtools.launch based upon package use.

Reply:
you're assumption is correct, I'll add a reference and comment.